spec/features/user_session_timeout_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'features/support/sign_up_helpers'
+require 'features/support/not_signed_in'
+require 'timecop'
+
+describe 'with devise timeoutable enabled' do
+  after { Timecop.return }
+
+  context 'when user has not signed in' do
+    before { visit root_path }
+
+    it_behaves_like 'not signed in'
+  end
+
+  describe 'user sessions timeout' do
+    context 'when user has signed in' do
```

Comment:
Given you don't test this context without combining it with the one on [line 23](https://github.com/alphagov/govwifi-admin/pull/277/files#diff-fa5dbd2027b8fe84348baeaa0a15255bR23), I would just have one describe block for the stuff between line 15 and line 37, something like
```ruby
context 'when a logged in user has been inactive for 60 minutes'
```